media-libs/tse3/files/tse3-0.2.7-gcc4.patch

```diff
@@ -1,5 +1,5 @@
---- ./src/tse3/cmd/Phrase.h~	2002-07-28 17:17:11.000000000 +0200
-+++ ./src/tse3/cmd/Phrase.h	2005-07-29 15:45:03.000000000 +0200
+--- a/src/tse3/cmd/Phrase.h	2002-07-28 17:17:11.000000000 +0200
++++ b/src/tse3/cmd/Phrase.h	2005-07-29 15:45:03.000000000 +0200
```

Comment:
also, in general remove the dates too, the just increase the noise in patches and arent useful